data/json/npcs/isolated_road/isolated_road_jay_dialogue.json

```diff
@@ -0,0 +1,243 @@
+            {
+              "u_compare_var": "gunsmith_ammo_ammount",
+              "type": "number",
+              "context": "artisans",
+              "op": "==",
+              "value": 800
+            },
+            { "u_has_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" }
+          ]
+        },
+        "topic": "TALK_GUNSMITH_BULLET_PICKUP"
+      },
+      {
+        "text": "I'd like to exchange some rounds.",
+        "condition": {
+          "and": [
+            { "u_has_var": "gunsmith_knows_about_conversion", "type": "dialogue", "context": "artisans", "value": "yes" },
+            {
+              "not": { "u_has_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" }
+            }
+          ]
+        },
+        "topic": "TALK_BULLET_EXCHANGE"
+      },
+      {
+        "text": "Are you and Cody close?",
+        "condition": {
+          "and": [
+            { "u_has_var": "gunsmith_mentioned_cody", "type": "dialogue", "context": "artisans", "value": "yes" },
+            {
+              "not": { "u_has_var": "gunsmith_accepted_quest", "type": "dialogue", "context": "artisans", "value": "yes" }
+            }
+          ]
+        },
+        "topic": "TALK_GUNSMITH_CODY"
+      },
+      { "text": "What is this place?", "topic": "TALK_GUNSMITH_LOCATION_ABOUT" },
+      { "text": "I have to go.", "topic": "TALK_DONE" },
+      {
+        "text": "About those contracts…",
+        "topic": "TALK_MISSION_LIST_ASSIGNED",
+        "condition": "has_many_assigned_missions"
+      },
+      {
+        "text": "Any jobs you need done?",
+        "condition": {
+          "and": [
+            { "u_has_var": "artisans_made_up", "type": "dialogue", "context": "artisans", "value": "yes" },
+            { "and": [ { "not": "has_assigned_mission" }, { "not": "has_many_assigned_missions" } ] }
+          ]
+        },
+        "topic": "TALK_MISSION_LIST"
+      },
+      { "text": "I'd like to make a withdrawl.", "topic": "TALK_GUNSMITH_SERVICES", "effect": "start_trade" }
+    ]
+  },
+  {
+    "id": "TALK_GUNSMITH_SHOP_ABOUT",
+    "type": "talk_topic",
+    "dynamic_line": [ "Fuck yeah.  New currency is death and I'm dealing it." ],
+    "responses": [
+      { "text": "I understand that but how does this work exactly?", "topic": "TALK_GUNSMITH_SHOP_ABOUT_MORE" },
+      { "text": "…", "topic": "TALK_GUNSMITH_SERVICES" }
+    ]
+  },
+  {
+    "id": "TALK_GUNSMITH_SHOP_ABOUT_MORE",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Pretty simple stuff bud'.  I barter in pre-shitshow ammo and to stimulate the growing economy print and issue more.  I take your goods, Cody would call it 'labor' and exchange it for the new universal currency.  If you want to convert currency I'm happy to facilitate but it'll take a few days and there is a service charge."
```

Comment:
I'm unsure about the "Cody would call it 'labor'" part, but not certain how to rephrase it.